README.rst

```diff
@@ -92,6 +92,39 @@ The library is tested with these specific, common types:
 
 * ``code39`` - `Code 39`_
 
+Command-line interface
```

Comment:
Use the header style above this one, add a line of `-` above the title too

Reply:
Gotcha. I thought it should go as a sub-section of API, but I'll change it to a top-level section.